src/nwb/NWBFile.cpp

```diff
@@ -50,14 +50,41 @@ NWBFile::~NWBFile() {}
 Status NWBFile::initialize(const std::string& identifierText,
                            const std::string& description,
                            const std::string& dataCollection)
+{
+  std::string time = getCurrentTime();
+  return this->initialize(
+      identifierText, description, dataCollection, time, time);
+}
+
+Status NWBFile::initialize(const std::string& identifierText,
+                           const std::string& description,
+                           const std::string& dataCollection,
+                           const std::string& sessionStartTime,
+                           const std::string& timestampReferenceTime)
```

Comment:
```suggestion
                           const std::string& timestampsReferenceTime)
```

Very minor and I don't think we need to change necessarily, but it's defined as `timestamps_reference_time` with an s in the file if we want to be consistent with naming

Reply:
Done